backend/sozisel/lib/sozisel_web/schema/queries/event_queries.ex

```diff
@@ -11,5 +11,12 @@ defmodule SoziselWeb.Schema.Queries.EventQueries do
       middleware Middleware.ResourceAuthorization, {:query_event, Event}
       resolve &EventResolvers.get_event/3
     end
+
+    field :event_result_details, strong_list_of(:event_result) do
```

Comment:
This query doesn't aggregate any data, so I had to do all of the parsing and aggregation on the frontend part. I think it will be better to do it on the backend side. I suggest to create following queries:
1. `quizEventSummary` which will return: `numberOfParticipants`, `averagePoints`, `averageAnswerTime`
2. `quizEventParticipantsResult` which will return for each participant: `numberOfPoints`, `answerTime`, and `participantQuizAnswer` for each question
3. `quizEventQuestionsResult` which will return for each question: `averagePoint`, `averageAnswerTime` and `participantQuizAnswer` for each participant

Reply:
done